server/src/main/java/org/apache/druid/server/lookup/cache/LookupLoadingSpec.java

```diff
@@ -39,6 +43,10 @@
  */
 public class LookupLoadingSpec
 {
+
+  public static final String CTX_LOOKUP_LOADING_MODE = "lookupLoadingMode";
+  public static final String CTX_LOOKUPS_TO_LOAD = "lookupsToLoad";
```

Comment:
I see a few context keys here-and-there ; but I think it would be better to: co-locate them somewhere
 (or introduce an infra to be pluggable/checkable/etc)

Could these context keys be moved to `QueryContexts` instead (that class is under `processing`) ?

Reply:
@kgyrtkirk Technically these constants aren't for query contexts, so `QueryContexts` class doesn't seem like a right place to move these to.

Reply:
Yeah, I am not too comfortable with these constants here myself. I had proposed moving them to `Tasks` but that doesn't seem viable either.

I guess we will have to live with them being in `LookupLoadingSpec` itself at least for the time being as there doesn't seem to be a cleaner alternative at the moment.